src/components/Tab/Tab.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import cn from 'classnames'
+
+const tabData = ['domains', 'bids', 'xxxxx']
```

Comment:
what's this?

Reply:
This is the tab component. For now set the list as static, once going to make it dynamic when I'm working on integrating.